crates/sui-config/src/genesis.rs

```diff
@@ -48,7 +48,12 @@ impl Genesis {
         let voting_rights = self
             .validator_set()
             .iter()
-            .map(|validator| (validator.public_key(), validator.stake()))
+            .map(|validator| {
+                (
+                    validator.public_key(),
+                    validator.stake() + validator.delegation(),
```

Comment:
Might make sense to add a single `ValidatorSet::voting_rights(&self) -> BTreeMap<PublicKey, u64>` utility function, since it looks we're doing this logic in three different places.